### PR TITLE
IR-20 Added word

### PR DESCRIPTION
**Pull request by reifred**
- The word was test
-

### CODE REVIEW

File: src/org/pahappa/systems/views/IReporter.java [hunk 1]
```diff
@@ -3,7 +3,7 @@
 public class IReporter {
 	
 	public static void main(String[] args) {
-		System.out.println("iReporter");
+		System.out.println("iReporter test");
```
Comment (reifred) [inline src/org/pahappa/systems/views/IReporter.java:6]: @reifred the word shouldn't be that

**Comment by reifred** [comment]
Change the word

File: src/org/pahappa/systems/views/IReporter.java (hunk 1 above)
Comment (reifred) [reply src/org/pahappa/systems/views/IReporter.java:6]: dfddggdgdgd

Comment (reifred) [reply src/org/pahappa/systems/views/IReporter.java:6]: sgsgss

**Review by MutebiAlexKato — APPROVED** [review]

**Pull request merged**